src/storage/v2/inmemory/label_property_index.hpp

```diff
@@ -136,6 +136,8 @@ class InMemoryLabelPropertyIndex : public storage::LabelPropertyIndex {
  private:
   std::map<std::pair<LabelId, PropertyId>, utils::SkipList<Entry>> index_;
   std::map<std::pair<LabelId, PropertyId>, storage::LabelPropertyIndexStats> stats_;
+
+  std::map<LabelId, std::set<PropertyId>> index_exists_;
```

Comment:
some more appropriate name, index_exists_ more sounds like a method than the description of what is it, e.g. property_indices_by_label or similar

Reply:
Done